packages/Jms/tests/Rector/Property/JmsInjectAnnotationRector/JmsInjectAnnotationRectorTest.php

```diff
@@ -0,0 +1,30 @@
+<?php declare(strict_types=1);
+
+namespace Rector\Jms\Tests\Rector\Property\JmsInjectAnnotationRector;
+
+use Iterator;
+use Rector\Testing\PHPUnit\AbstractRectorTestCase;
+
+/**
+ * @covers \Rector\Jms\Rector\Property\JmsInjectAnnotationRector
+ */
+final class JmsInjectAnnotationRectorTest extends AbstractRectorTestCase
```

Comment:
there is `JmsRectorTest`
combine these?

Reply:
I think so, since only this one will be used.

Should be here fallback to @var annotation in case of missing named service?
Again, docs would help

Reply:
Added